MainGame.py

```diff
@@ -25,21 +26,45 @@ def Turn(self, player: Player) -> None:
 
             if isvalidTurn:
                 self.board.AddCoinToBoard(row, player.symbol)
-                return
 
-            print(
-                f"Du kannst deinen Stein nicht in Spalte '{row}' setzen. Versuch es nochmal!"
-            )
+                playerWins = StatusValidator.isWin(self.board, player, row)
+
+                if playerWins:
+                    return player
+                break
+
+            else:
+                print(
+                    f"Du kannst deinen Stein nicht in Spalte '{row}' setzen. Versuch es nochmal!"
+                )
 
-    def Round(self) -> None:
+                return
+
+    def Round(self) -> None or Player:
+        winner: Player
         self.roundNumber += 1
+        maxRounds = 21
+        if self.roundNumber > maxRounds:
+            return Player()
         print(f"Round: {self.roundNumber}")
-        self.Turn(self.playerOne)
-        self.Turn(self.playerTwo)
+        winner = self.Turn(self.playerOne)
+        if winner is None:
+            winner = self.Turn(self.playerTwo)
+
+        if winner is not None:
+            return winner
```

Comment:
It would make more sense to move the "is player the winner?" variable to the `Player` class